docs/advanced_usage.rst

```diff
@@ -241,6 +241,46 @@ next step of the pipeline or ``None`` if the trace should be discarded::
 
 (see filters.py for other example implementations)
 
+.. _`Logs Injection`:
```

Comment:
Great as a first pass, but maybe a bit hidden in our Advanced Usage. For now let's land this section here, then we must put something there: https://docs.datadoghq.com/tracing/advanced_usage/?tab=python#logging

Reply:
https://github.com/DataDog/documentation/pull/3726